app/Http/Requests/category/CreateCategoryRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Requests\Category;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateCategoryRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|min:3|max:255|unique:categories'
+        ];
+    }
+
+    public function messages()
+    {
+        return [
+            'name.unique' => 'Name is already',
```

Comment:
```suggestion
            'name.unique' => trans('name.unique'),
```
M suggest đặt key và message ntn nha :3